source/_integrations/cloudflare.markdown

```diff
@@ -40,15 +40,80 @@ api_key:
   required: true
   type: string
 zone:
-  description: The DNS zone you want to update.
+  description: The DNS zone (domain) you want to update.
   required: true
   type: string
 records:
-  description: A list of records you want to update.
+  description: A list of records (subdomains) you want to update.
   required: true
   type: list
 {% endconfiguration %}
 
+## Additional information
+
+### Usage of external service
+
 This platform uses the API from [ipify.org](https://www.ipify.org/) to set the public IP address.
 
+
+### API Key
+
+Please note that the `api_key` is the [global API key](https://support.cloudflare.com/hc/en-us/articles/200167836-Managing-API-Tokens-and-Keys#12345682) of your Cloudflare account (not the API Token).
+
+
+### Home Assistant Companion App
+
+If you would like to use [iOS App](https://companion.home-assistant.io/) via Cloudflare set **Minimum TLS version as 1.2**, in order to do that, do the following:
+1. Login to your [Cloudflare](https://dash.cloudflare.com/) account.
+2. Choose your domain.
+3. Click on icon `SSL/TLS`.
+4. Go to tab `Edge Certificates`.
+5. Find `Minimum TLS Version` and set it to **1.2**.
+
+Other settings should not cause any issues.
+
+
+### SSH over Cloudflare
+
 For SSH usage (according to [this](https://blog.cloudflare.com/cloudflare-now-supporting-more-ports/) source), you need to connect directly to your server (bypassing Cloudflare). To do that create a `CNAME` DNS record, e.g., `ssh.example.com`, with proxy status as "DNS only" (to do that click on orange icon, it will change color to gray) and then connect to `ssh.example.com` using your server SSH port.
+
+
+### Using Cloudflare domain only for Home Assistant
+
+If you want to update just a main domain, place in the records list only your domain, e.g. `example.com`. It will update your `A` DNS record with your IP every hour.
```

Comment:
```suggestion
If you want to update just a main domain, place in the records list only your domain, e.g., `example.com`. It will update your `A` DNS record with your IP every hour.
```